draft-ietf-quic-tls.md

```diff
@@ -190,15 +192,10 @@ TLS connection for use by QUIC.  This keying material is used to derive packet
 protection keys.  Details on how and when keys are derived and used are included
 in {{packet-protection}}.
 
-This arrangement means that some TLS messages receive redundant protection from
-both the QUIC packet protection and the TLS record protection.  These messages
-are limited in number; the TLS connection is rarely needed once the handshake
-completes.
```

Comment:
I'd kinda like to keep something like this.  We still double-encrypt NST and people will complain (or get it wrong) if we don't explain it somewhere.